common/persistence/sql/sqlTaskManager.go

```diff
@@ -170,50 +170,35 @@ func (m *sqlTaskManager) UpdateTaskQueue(request *persistence.UpdateTaskQueueReq
 	}
 
 	tqId, tqHash := m.taskQueueIdAndHash(nidBytes, request.TaskQueueInfo.Name, request.TaskQueueInfo.TaskType)
-
 	tq := request.TaskQueueInfo
 	tq.LastUpdateTime = timestamp.TimeNowPtrUtc()
 
-	var blob commonpb.DataBlob
+	var resp *persistence.UpdateTaskQueueResponse
 	if request.TaskQueueInfo.Kind == enumspb.TASK_QUEUE_KIND_STICKY {
 		tq.ExpiryTime = stickyTaskQueueTTL()
-		blob, err = serialization.TaskQueueInfoToBlob(tq)
-		if err != nil {
-			return nil, err
-		}
-		if _, err := m.db.ReplaceIntoTaskQueues(&sqlplugin.TaskQueuesRow{
```

Comment:
Is this function used anymore? Can we remove it and its corresponding implementations?